packages/frontend/src/components/chart/configure/persistedSettings.ts

```diff
@@ -0,0 +1,35 @@
+import { State } from './state/State'
+
+interface PersistedSettings {
+  isLogScale: boolean
+}
+
+const DEFAULT_SETTINGS = {
+  isLogScale: false,
+}
+
+export function persistSettings(state: State) {
```

Comment:
there is no need to pass the whole State object

Reply:
Not now, but I'll implement other settings like days and currency soon. Do you prefer to pass them explicitly rather than the whole State?